src/moscot/_docs/_docs.py

```diff
@@ -134,6 +134,26 @@
     Specifies the right marginals. If of type :class:`str` the right marginals are taken from
     :attr:`anndata.AnnData.obs` ``['{b}']``. If `b` is `None` uniform marginals are used.
 """
+_a_temporal = """\
+a
+    Specifies the left marginals. If
+        - ``a`` is :class:`str` - the left marginals are taken from :attr:`anndata.AnnData.obs`,
+        - if :meth:`~moscot.problems.base._birth_death.BirthDeathMixin.score_genes_for_marginals` was run and
+        if `a` is `None`, marginals are computed based on a birth-death process as suggested in
+        :cite:`schiebinger:19`,
+        - if :meth:`~moscot.problems.base._birth_death.BirthDeathMixin.score_genes_for_marginals` was run and
+        if `a` is `None`, and additionally `c` is provided in `marginal_kwargs`, the marginals are computed as
```

Comment:
Can we give it a more descriptive name than `c`?

Reply:
called it `scaling` now